src/api.js

```diff
@@ -280,6 +280,10 @@ api.declare({
   msg.repository = sanitizeGitHubField(body.repository.name);
   msg.eventId = eventId;
 
+  msg.created=new Date();
+  var start = msg.created.getTime(); 
+
+
```

Comment:
We should remove this extra empty line. Just one empty line at a time to make our lint happy. Try running `yarn lint` and seeing if it says anything else is awry.